### DISCUSSION TITLE
do I need @vueform/builder to render a form?

### DISCUSSION THREAD

**Discussion by julisch94** [vueform/vueform#110] [Questions] [answered]
hey folks! thanks for creating this library. I'm currently trying to set up vueform for Vue.js and Tailwind but I'm having troubles.

first question I have is this:
In the docs (https://vueform.com/docs/installation#manual-installation) I can see that a `builderConfig` is imported as well as @vueform/builder but it's not mentioned in the other steps. Do I need that builder library? And where do I get the config from? Thanks!

```ts
// path/to/main.(js|ts)

import { createApp } from 'vue'
import App from './components/App.vue'
import Vueform from '@vueform/vueform'
import vueformConfig from './path/to/vueform.config'
import Builder from '@vueform/builder'
import builderConfig from './../../builder.config'
// ...

const app = createApp(App)
app.use(Vueform, vueformConfig)
app.use(Builder, builderConfig)
app.mount('#app') 
 ```

**Comment by adamberecz** [comment, marked as answer]
hey @julisch94! No, you don't need to builder to render form. Are you sure you were checking the right docs [here](https://vueform.com/docs/installation#manual-installation) and not on our builder installation page (which is very similar)?

**Reply by julisch94** [reply]
Thanks @adamberecz! that's good to know. I'll leave it out then :)

I just double-checked and I can confirm that the docs, in fact, contain those lines:

1. open https://vueform.com/docs/installation
2. select Vue.js
3. select Tailwind
4. you'll then see the snippet that I posted above containing imports of builder

If the docs are open-source, I'm happy to open a PR for it.

**Reply by adamberecz** [reply]
Thanks for spotting the issue getting back on it @julisch94 - got it fixed.
